src/lib/components/edit-client.svelte

```diff
@@ -89,6 +104,17 @@
 	function confirmDeleteClient() {
 		showConfirmModal = true
 	}
+
+	$effect(() => {
+		const res = emailFormSchema.safeParse({ email })
+		if (res.success) {
+			emailError = undefined
+			emailValid = true
```

Comment:
Not sure this is needed but it probably should be here
```suggestion
			emailError = undefined
			emailValid = true
			emailTouched = false
```